### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by danmichaelo** [ShowMeHills/ShowMeHills#25]
The [readme](https://github.com/ShowMeHills/ShowMeHills/blob/master/README) currently says "The database of mountains is limited to the UK", but it seems like this is no longer the case?

Including some info to get potential contributors started would also be great! Such as build instructions.

**Comment by nikcain** [comment]
Yes, I totally agree - it's got a bit easier to build it after I moved it to Android Studio (which the readme doesn't mention, so that would have helped!), but modifying the database would probably be impossible for anyone to do with asking me directly. 

So I promise to improve it!
